mini-frame.el

```diff
@@ -92,6 +92,10 @@ to determine background color of mini frame."
 Called if `mini-frame-show-parameters' doesn't specify background color."
   :type 'function)
 
+(defcustom mini-frame-border-color "gray50"
+  "MiniFrame border color."
+  :type 'string)
```

Comment:
Use `color` instead of `string` for type.

Reply:
done